crates/matrix-sdk-ui/src/timeline/controller/mod.rs

```diff
@@ -604,11 +604,18 @@ impl<P: RoomDataProvider> TimelineController<P> {
 
     /// Is this timeline focused on a thread?
     pub(super) fn is_threaded(&self) -> bool {
-        matches!(&*self.focus, TimelineFocusKind::Thread { .. })
+        self.focus.is_thread()
     }
 
+    /// The root of the current thread, for a live thread timeline or a
+    /// permalink to a thread message.
```

Comment:
> permalink to a thread message.

Is the idea here for `TimelineFocusKind::Event` to eventually handle threaded permalinks too?

Reply:
Exactly, you can read the full plan in https://github.com/matrix-org/matrix-rust-sdk/issues/5535